webpack.config.js

```diff
@@ -21,21 +27,59 @@ module.exports = (env) => {
       /\bwebpack\/buildin\b/
     ];
 
-    filename += '.bundle';
+    babelOptions = {
+      sourceType: 'unambiguous',
+      ignore: [/\/core-js/],
+      presets: [
+        ['@babel/preset-env',
+          {
+            debug: false, /* set to `true` if you start banging your head against the wall */
+            targets: [
+              'last 2 versions',
+              '> 5%',
+              'IE 11',
+              'not dead'
+            ]
+          }
+        ]
+      ],
+      plugins: [
+        ['@babel/plugin-transform-runtime', {
+          corejs: 3
+        }]
+      ]
+    };
   } else {
-    /* if we are building for 'module', don't polyfill/transpile any dependencies */
-    babelExcludes = [/node_modules/];
+    /* we are building for node */
+    const modulesToBundle = [/* left this here just in case it is needed */];
+
+    /* we don't want to bundle node_modules */
+    externals.push(nodeExternals({
+      allowlist: modulesToBundle.map(m => new RegExp(`^${m.replace(/-/g, '\\-')}`))
+    }));
```

Comment:
The `externals` along with [webpack-node-externals](https://www.npmjs.com/package/webpack-node-externals) was the main missing piece I couldn't figure out. This tells webpack to not bundle/build dependencies. This allows consumers to build and transpile the streaming-client using any node build tool they like (like lasso, webpack. rollup, ember, etc). 

The other missing was the `target: 'web'`. [See my other comment below](https://github.com/purecloudlabs/genesys-cloud-streaming-client/pull/64/files#r464675419)